manager-bundle/src/EventListener/DoctrineListener.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\ManagerBundle\EventListener;
+
+use Doctrine\DBAL\Event\SchemaAlterTableRenameColumnEventArgs;
+
+/**
+ * @internal
+ */
+class DoctrineListener
```

Comment:
I also don't really like this name, don't we already have several doctrine listeners that do different things? Should this be a `DoctrineAlterTableListener` ?

Reply:
Renamed in eb4acea2b951aa7545a38a72aa4120e1f880dec7